src/govuk/components/accordion/accordion.test.js

```diff
@@ -251,13 +251,14 @@ describe('/components/accordion', () => {
 
           const $element = await page.$('.govuk-accordion__section-button')
 
-          expect(getAccessibleName($element)).resolves.toBe(
+          await expect(getAccessibleName(page, $element)).resolves.toBe(
```

Comment:
Puppeteer was receiving the `.click()` concurrent to Jest resolving the promise

It was flagged by TypeScript in another PR so was my first guess at fixing the issue

* https://github.com/alphagov/govuk-frontend/pull/2987